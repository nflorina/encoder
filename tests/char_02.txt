R
F
=
]
M
H
R
o
V
U
v
\
L
b
B
B
s
)
.
)
v
I
$
{
&
V
i
+
s
y
:
g
J
q
q
g
B
$
'
[
$
d
J
|
-
{
W
r
z
c
F
v
\
w
y
W
&
q
{
{
h
(
>
F
Y
)
1
END
