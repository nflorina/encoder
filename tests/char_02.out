0 66 0
RRFRR=FR]=RM]RHMRRHRoFRVoRUVRvUR\vRL\RbLRBbRBBRs=R)sR.)R).Rv]RIMR$IR{$R&{RV&RiHR+iRs+RyoR:yRg:RJgRqJRqqRgURB\R$LR'bR['R$[Rd.RJdR|IR-|R{-RW&RrWRzrRczRFcRviR\+RwyRywRW:R&'Rq[R{dR{|Rh-R(hR>(RF>RYrR)Y
RRFRR=FR]=RM]RHMRRHRoFRVoRUVRvUR\vRL\RbLRBbRBBRs=R)sR.)R).Rv]RIMR$IR{$R&{RV&RiHR+iRs+RyoR:yRg:RJgRqJRqqRgURB\R$LR'bR['R$[Rd.RJdR|IR-|R{-RW&RrWRzrRczRFcRviR\+RwyRywRW:R&'Rq[R{dR{|Rh-R(hR>(RF>RYrR)Y
